packages/ui/src/ui/components/WarningIcon/WarningIcon.tsx

```diff
@@ -11,14 +13,21 @@ const b = cn('yt-warning-icon');
 interface Props {
     className?: string;
     hoverContent?: React.ReactNode;
+    color?: 'warning' | 'danger';
+    children?: React.ReactNode;
 }
 
 export default function WarningIcon(props: Props) {
-    const {className, hoverContent} = props;
+    const {className, hoverContent, children, color} = props;
     return (
         <span className={b(null, className)}>
             <Tooltip className={b('tooltip')} content={hoverContent}>
-                <Icon className={b('icon')} awesome="exclamation-triangle" />
+                <Icon className={b('icon', {color})} awesome="exclamation-triangle" />
```

Comment:
We don't want to slowly abandon the old components? Maybe we should replace it with an `Icon` from data-ui with this [icon](https://gravity-ui.com/icons?icon=TriangleExclamation).